README.md

```diff
@@ -100,6 +100,23 @@ grunt.initConfig({
 
 Since tail runs till you send it a shutdown signal, you would like to stream the output to your stdio.
 
+#### Progress indeicator
```

Comment:
spelling fixed, sorry for the new pull request, had a rebase misfire which auto closed this pull request